src/pybind/mgr/dashboard/controllers/user.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from . import ApiController, AuthRequired, RESTController
+from .. import mgr
+from .auth import Auth
+
+
+@ApiController('user')
+@AuthRequired()
+class User(RESTController):
+    # pylint: disable=unused-argument
+    def get(self, username):
+        config_username = mgr.get_config('username', None)
```

Comment:
I think the user management code here and in `controllers/auth.py` should be moved into a central place like the `services` directory, so that in the future if we change the key of the config, where we store the username and password, we don't need to look for all the places where it is used.